Dockerfile

```
@@ -1,12 +1,11 @@
 ARG APP_ENV=prod
 
 # Building builder image
-FROM alpine:latest as builder
+FROM eclipse-temurin:17-jre-focal as builder
```

:x: Please point it to `ubuntu:focal`. This will avoid downloading the JRE image when building, for example, the dev image (which uses the JDK variant). Plus, just running the builder stage (where we do most of the stuff anyway, and probably sees more changes) will be faster.

```suggestion
FROM ubuntu:focal as builder
```